slack-post-message/README.md

```diff
@@ -0,0 +1,55 @@
+# slack-post-message
+
+slack-post-message adds a "Post a message" button in the slack shortcuts section. This can be used to post a message in multiple channels. The message will be posted in the channels selected, by the bot on behalf of the user. Only the users part of the *slack usergroups* given in the config file are permitted to use the bot. This app uses [Slack modals](https://api.slack.com/surfaces/modals) rather than dialogs
+
+![screenshot of a blank send message modal](./screenshot.png)
+
+## Configuration
+
+slack-post-message requires a configuration file, by default called `config.json` in the working
+directory. It must look like this:
```

Comment:
This does not seem to reflect what we do in the deployment were we provide a argument ` --config-path=/etc/slack-post-message/config.json`

Reply:
Hey. I am sorry, but isn't it the right path in the argument, since the **config** volume is mounted at `/etc/slack-post-message`? Correct me if I am wrong

Reply:
It states here the config needs to be in the working directory but it does not need to be if you provide the right argument.